### PR TITLE
Fixup our README file a bit

### PR DESCRIPTION
**Pull request by stieg**
This nice commit cleans up our README file a lot so that we can
have a better intro page for those looking to contribute.  The
packages I mentioned my commit were ones loosely based off of a
fedora 21 install.

I suggest having a look at the markup.  Its much better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,61 @@
...
 
-================================
-Windows
-================================
+MK1 was the original RaceCapturePro unit.  Based on the AT91SAM7s chipset, it provided
+a solid platform for our initial release.  It allowd for up to a 100Hz channel sampling
+rate and up to 10Hz GPS.  It also supported CAN & OBD2, various I/O, an SD card, and
+a remote GPS mouse for high quality GPS reception.  It came with a separate telemetry
+unit that would handle passing the data over the Cellular network.  It however has now
+been succedded by the next generation of RaceCapturePro
```
Comment (ryandoherty) [inline README.md]: succeeded and RaceCapture/Pro

File: README.md [hunk 2]
```diff
@@ -1,39 +1,61 @@
...
-================================
-Linux
-================================
-While holding down the button on RaceCapture/Pro, plug the unit into USB. You should see 3 green LEDs light up.
+MK2 is the latest RaceCapturePro unit.  Like its successor it has many of the same features
+that were originally loved, all of which have been improved upon.  It also has some new features
+that were unavialable in the original MK1.  MK2 supports sampling rates of up to 1000 Hz per
+channel (a 10x improvement from MK1) and a GPS unit that can sample up to 50Hz (a 5x improvement
+from MK1).  It also includes a better Inertial unit, an upgraded processor (STM32 base), more RAM
+(for better LUA support) and an integrated cellular device (optinal).  The GPS unit is also integrated
```
Comment (ryandoherty) [inline README.md]: lowercase inertial and fix optional

File: README.md [hunk 3]
```diff
@@ -46,7 +68,67 @@ Mk2 Building Requirements
...
+### Compiling
+
+Follow these steps to compile the unit tests:
+
+* cd test
+* make
+
+### Testing
+
+Simply run `./rcptest` from within the 'test' directory.  That will run the suite of unit test.
```
Comment (ryandoherty) [inline README.md]: s/test/tests

**Comment by ryandoherty** [comment]
This is great! Much improved. 

Let's change all 'RaceCapturePro' to 'RaceCapture/Pro', which is the standardized spelling.

File: README.md [hunk 4]
```diff
@@ -1,39 +1,61 @@
-RaceCapture/Pro Firmware
-========================
+RaceCapturePro Firmware
+=====
 
-Firmware for RaceCapture/Pro data acqusition / control / telemetry system
+The premier open source telemetry system for your race vehicle.  RaceCapturePro is a
+device that gets installed into your race vehicle and beams telemetry about your
+vehicle and driving habits in real time.  Combined with
+[RaceCapture App](https://github.com/autosportlabs/RaceCapture_App) and
```
Comment (brentpicasso) [inline README.md]: Suggest: 

A powerful and flexible open source telemetry system for motorsport applications. RaceCapture/Pro installs into your race vehicle and transmits real-time wireless telemetry data via GSM cellular and Bluetooth.  Combined with

File: README.md [hunk 5]
```diff
@@ -1,39 +1,61 @@
...
 
-WARNING: Updating firmware restores the default configuration on RaceCapture/Pro
-Be sure to save your configuration using the Race Analyzer software.
+## MK1
 
-================================
-Windows
-================================
+MK1 was the original RaceCapturePro unit.  Based on the AT91SAM7s chipset, it provided
+a solid platform for our initial release.  It allowd for up to a 100Hz channel sampling
```
Comment (brentpicasso) [inline README.md]: RaceCapture/Pro

File: README.md [hunk 6]
```diff
@@ -1,39 +1,61 @@
...
+rate and up to 10Hz GPS.  It also supported CAN & OBD2, various I/O, an SD card, and
+a remote GPS mouse for high quality GPS reception.  It came with a separate telemetry
```
Comment (brentpicasso) [inline README.md]: Suggest: 

.... It also supported CAN and OBD-II (via external MCP2515 CAN controller), analog and digital sensors, an SD card ...

File: README.md (hunk 1 above)
Comment (brentpicasso) [inline README.md]: RaceCapture/Pro

File: README.md [hunk 7]
```diff
@@ -1,39 +1,61 @@
...
 
-================================
-Linux
-================================
-While holding down the button on RaceCapture/Pro, plug the unit into USB. You should see 3 green LEDs light up.
+MK2 is the latest RaceCapturePro unit.  Like its successor it has many of the same features
+that were originally loved, all of which have been improved upon.  It also has some new features
+that were unavialable in the original MK1.  MK2 supports sampling rates of up to 1000 Hz per
+channel (a 10x improvement from MK1) and a GPS unit that can sample up to 50Hz (a 5x improvement
+from MK1).  It also includes a better Inertial unit, an upgraded processor (STM32 base), more RAM
```
Comment (brentpicasso) [inline README.md]: a GPS module capable of native 50Hz update rate....

..It also includes an improved 9 axis Inertial Motion Unit....

...an upgraded processor (STM32F407 Cortex M4F)....

**Comment by brentpicasso** [comment]
This is fantastic, thank you for fleshing this out!

**Comment by stieg** [comment]
Redid this significantly so going to open a new req.

**Pull request closed without merging**